tests/textcase/plugin/plugin_spec.lua

```diff
@@ -12,10 +12,6 @@ end
 
 describe("plugin", function()
   before_each(function()
-    -- Init has to be called in the plugin tests because in normal usage it is
-    -- loaded in the plugin/start.vim file
-    textcase.init()
```

Comment:
[This specific piece of code](https://github.com/johmsalas/text-case.nvim/blob/804e4f9049df3eaa3f9611e28f80a75f1d9e023b/tests/environments/minimal.lua#L14) loads the vim file

Reply:
Ah good to know! Thanks :) I was already wondering what I was missing when I added the tests.